{{ cookiecutter.project_name }}/bin/run-dev.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 
+urlwait  # Wait for the database to come online
```

Comment:
It defaults to `DATABASE_URL`, but you can [use the CLI](https://github.com/pmclanahan/urlwait#usage) to specify a URL or a different variable to watch.